### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by lanthias**
New community-focused GitHub readme with updated examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,43 @@
...
+    <img src="https://user-images.githubusercontent.com/3541695/176245723-3874bf39-b2c3-42e0-802c-6d8176e7b7d2.png"  width='75%'>
+  </a>
+</p>
 
-Import our library into your existing script/notebook and build reports from pandas Dataframes, plots from Python viz libraries, Markdown, as well as data exploration and layout components.
+### What makes Datapane special?
 
-Export your reports as standalone HTML documents, or share and embed them via our free hosted platform.
+- **Static generation:** Sharing a report shouldn't require deploying an app. Render a standalone HTML bundle which you can share or host on the web.
+- **API-first and programmatic:** Programmatically generate reports from inside of Spark, Airflow, or Jupyter. Schedule updates to build realtime dashboards.
```
Comment (shahinrostami) [inline README.md:42]: realtime -> real-time

File: README.md [hunk 2]
```diff
@@ -45,164 +58,137 @@ $ datapane hello-world
...
 ```
 
-## Explainer Video
+# Creating reports
 
-https://user-images.githubusercontent.com/16949044/134007757-0b91074a-2b32-40ba-b385-5623dff8c04e.mp4
+### 📊 Include plots and data
 
-## Hello world
+Create a report from pandas DataFrames, plots from your favourite libraries, and text.
```
Comment (shahinrostami) [inline README.md:75]: favourite -> favorite

File: README.md [hunk 3]
```diff
@@ -4,30 +4,43 @@
...
+</p>
+Datapane is an open-source framework makes it easy to create beautiful reports from anywhere you can run Python.
+Love analyzing data in Python but struggle to share results and insights? Datapane is for you!
+<br>
+<br>
+<br>
+
+<p align="center">
+  <a href="https://datapane.com">
+    <img src="https://user-images.githubusercontent.com/3541695/176245723-3874bf39-b2c3-42e0-802c-6d8176e7b7d2.png"  width='75%'>
```
Comment (shahinrostami) [inline README.md:35]: This image is front and centre – shall we make a better one?

File: README.md [hunk 4]
```diff
@@ -4,30 +4,43 @@
...
-    <a href="https://pypi.org/project/datapane/">
-        <img src="https://img.shields.io/pypi/dm/datapane?label=pip%20downloads" alt="Pip Downloads" />
-    </a>
-    <a href="https://pypi.org/project/datapane/">
-        <img src="https://img.shields.io/pypi/v/datapane?color=blue" alt="Latest release" />
-    </a>
-    <a href="https://anaconda.org/conda-forge/datapane">
-        <img alt="Conda (channel only)" src="https://img.shields.io/conda/vn/conda-forge/datapane">
-    </a>
+  <a href="https://datapane.com">Datapane.com</a> |
```
Comment (shahinrostami) [inline README.md:7]: When I visit a repo my eyes start looking for “Examples” or “Demo” – shall we link directly to our showcase here?

File: README.md [hunk 5]
```diff
@@ -45,164 +58,137 @@ $ datapane hello-world
...
 
-Your Studio account comes with the following:
-
-- **Unlimited public reports** - great for embedding into places like Medium, Reddit, or your own website (see [here](https://docs.datapane.com/reports/embedding-reports-in-social-platforms))
-- **5 private reports** - share these via email within your organization
-
-### Featured Examples
+# Demos and Examples
 
 Here a few samples of the top reports created by the Datapane community. To see more, check out our [gallery](https://datapane.com/gallery) section.
```
Comment (shahinrostami) [inline README.md:177]: the gallery link https://datapane.com/gallery redirects me to https://datapane.com

File: README.md [hunk 6]
```diff
@@ -45,164 +58,137 @@ $ datapane hello-world
...
 
-- [Tutorial Report](https://datapane.com/u/leo/reports/tutorial-1/) by Datapane Team
 - [Coindesk analysis](https://datapane.com/u/greg/reports/initial-coindesk-article-data/) by Greg Allan
 - [COVID-19 Trends by Quarter](https://datapane.com/u/keith8/reports/covid-19-trends-by-quarter/) by Keith Johnson
 - [Ecommerce Report](https://datapane.com/u/leo/reports/e-commerce-report/) by Leo Anthias
```
Comment (shahinrostami) [inline README.md:181]: Overlapping text in the figures

Screenshot 2022-06-29 at 17 48 55

File: README.md (hunk 3 above)
Comment (lanthias) [reply README.md:35]: yeah do we have a better one? alternatively, we can have this as a placeholder and once the video is done we could put that there.

Comment (shahinrostami) [reply README.md:35]: Just put this together - what do you think?

new_dp_promo

**Review by shahinrostami — APPROVED** [review]
Looking good!

**Comment by antdking** [comment]
Sorry to hijack; just making the linter happy as I know you've stuff on at the moment :)

**Pull request merged**